radixdlt/src/main/java/org/radix/Radix.java

```diff
@@ -71,6 +77,22 @@
 	private static final Object BC_LOCK = new Object();
 	private static boolean bcInitialised;
 
+	// Be careful here.  This is a debugging method to detect overcommitted heap usage on Linux.
```

Comment:
This should not be here.  Need to rebase on current head of `feature/rpnv1-67-oom-killer-linux`